src/components/organisms/NavBarSchedule/NavBarSchedule.scss

```diff
@@ -16,16 +16,22 @@ $schedule-btn--height: 51px;
   flex-direction: column;
 
   &__download-buttons {
+    width: $NavBarSchedule--width;
     position: fixed;
     bottom: 0;
-    width: 40%;
     z-index: z(navbar-schedule-download);
     display: flex;
     align-items: center;
     justify-content: flex-end;
     background-color: opaque-black(0.9);
     height: $schedule-btn--height;
     padding-left: $NavBarSchedule--padding;
+    transform: translateX(-$NavBarSchedule--width);
+
+    &--show {
+      transition: all 400ms $transition-function;
```

Comment:
please, move `transition` from `&--show` to a `&__download-buttons`. It'll make the transition smoother.